### ISSUE TITLE
Trans-species learning of cellular signaling systems with bimodal deep belief networks

### ISSUE THREAD

**Issue by cgreene** [greenelab/deep-review#9]
Paper needs to be read carefully for relevance.
https://dx.doi.org/10.1093/bioinformatics/btv315

**Comment by hussius** [comment]
Interesting as one of a small number of examples of something resembling "transfer learning" for biology.

**Comment by cgreene** [comment]
@hussius : definitely! I am highly interested in the potential for transfer learning. In some ways the simultaneous optimization of multiple motifs in #13 is a bit transfer learning-ish also. It's learning general properties of features that may be relevant for multiple problems.

**Comment by cgreene** [comment]
Computational:

This paper includes an interesting transfer learning approach to deal with both rat and human phosphoproteomics data. Interactions are allowed between proteins in the same species and disallowed between different species. The goal is then to learn the structure with a training set, and to predict the human protein states when provided only rat proteins on a test set.

One thing that's kind of interesting in the results - the precision of the SVM at low recall is pretty good. The DBN is actually not that good, but the sbDBN is better than SVM. In AUC space, the DBM and sbDBN are both better than SVM.

Bio:

Interesting cross-species prediction of phosphoproteome states. Question centers around how signalling is maintained between a model and target system. They do some biological interpretation of the edges in the sbDBN model by looking at the strongest edge predictions.

Caveats:

Well, there's only 234 examples and few features. It's not the traditional space for deep learning. It also looks like they trained 120 models for the networks and only reported the 'best' according to AUROC (section 3.2). This might render comparisons with other methods somewhat challenging. It's also a bit hard to compare across the two DBN methods because the network structure is different.

Also, as I'm reading this there are multiple endpoints. Specifically the authors need to predict whether each human protein is phosphorylated. The AUROC reported is only for one.

Despite the caveats of this specific instance, I think we would be remiss not to discuss transfer learning. I think it'd be helpful to talk about this, even with all its caveats - though we may want to be careful to stay away from using the actual performance number/comparison from the paper to argue something positive about DBNs & transfer learning.

**Comment by cgreene** [comment]
As a positive: source code is available [matlab]

**Comment by cgreene** [comment]
I think we really need to talk about transfer learning in bio. This paper probably fits most naturally in 'study'. I'm also tagging it for 'categorize' in case we want to discuss there as well, but that's a looser fit.

**Comment by bios-transspecies** [comment]
oh... thats my topic :) https://github.com/bios-transspecies/Transspecies-Blogging
